io.catenax.mp_standard_response/1.0.0/DataTypes.ttl

```diff
@@ -0,0 +1,53 @@
+##########################################################################################
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e. V.
+# Copyright (c) 2023 Siemens AG
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.mp_standard_unsubscribe:1.0.0#> .
+@prefix ext-header: <urn:samm:io.catenax.mp_standard_datatypes:1.0.0#> .
+
+:timeUnit a samm:Property ;
+   samm:preferredName "timeUnit"@en ;
+   samm:description "specifies the unit in which the time es represented"@en ;
+   samm:characteristic :TimeUnitEnum ;
+   samm:exampleValue "Month" .
+
+:TimeUnitEnum a samm-c:Enumeration ;
+   samm:preferredName "TimeUnitEnum"@en ;
+   samm:description "Enumerates all possible time units"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Month" "Day" "Week" "Hour" "Minute" "Second" ) .
+
+:value a samm:Property ;
+   samm:preferredName "value"@en ;
+   samm:description "the amount of timeUnits considered"@en ;
+   samm:characteristic :Value ;
+   samm:exampleValue 5 .
+
+:TimeValue a samm:Entity ;
+   samm:preferredName "TimeValue"@en ;
+   samm:description "Datatype to express a time value"@en ;
+   samm:properties ( :timeUnit :value ) .
+
+:Value a samm:Characteristic ;
+   samm:preferredName "Value"@en ;
+   samm:description "the value of the specified timeUnit"@en ;
+   samm:dataType xsd:integer .
+
```

Comment:
As I commented in the other model, I am not sure, if the data structures in this model could not be replaced by the timestamp characteristic